cvat/apps/engine/cloud_provider.py

```diff
@@ -761,17 +765,13 @@ def _list_raw_content_on_one_page(
 
     @validate_file_status
     @validate_bucket_status
-    def download_fileobj(self, key: str) -> NamedBytesIO:
-        buf = NamedBytesIO()
+    def _download_fileobj_to_stream(self, key: str, stream: BinaryIO) -> None:
         storage_stream_downloader = self._client.download_blob(
             blob=key,
             offset=None,
             length=None,
         )
-        storage_stream_downloader.download_to_stream(buf, max_concurrency=self.MAX_CONCURRENCY)
-        buf.seek(0)
-        buf.filename = key
-        return buf
+        storage_stream_downloader.download_to_stream(stream, max_concurrency=self.MAX_CONCURRENCY)
```

Comment:
The `download_to_stream` method is deprecated, could you please update to `readinto`? 
```suggestion
        storage_stream_downloader.readinto(stream)
```
and pass `max_concurrency`  in the call above
https://github.com/Azure/azure-sdk-for-python/blob/735259d15bcd7b122fae01a29dd41a36a882c578/sdk/storage/azure-storage-blob/azure/storage/blob/_download.py#L909
(It isn't really related to the PR, but probably can be fixed there)

Reply:
done